windows.md

```diff
@@ -545,14 +545,14 @@ With those compatibility things out of the way, you're ready to start the system
 
     Click on the small `x` at the top right of the virtual device frame to stop the virtual device - this will save a snapshot to make starting the virtual device faster in the future.
 
-25. If you don't have one yet, create a Google account [here](https://accounts.google.com/signup?hl=en). Make a note of the email address associated with this account for usage in later steps.
-26. If you don't have one yet, create a GitHub account [here](https://github.com/join). Make sure to set a name.
+24. If you don't have one yet, create a Google account [here](https://accounts.google.com/signup?hl=en). Make a note of the email address associated with this account for usage in later steps.
+25. If you don't have one yet, create a GitHub account [here](https://github.com/join). Make sure to set a name.
 
     If you already have a GitHub account and you haven't set a name on GitHub yet, go to the [GitHub Profile Settings](https://github.com/settings/profile) and add a name:<br><br>
     <img src="./general-github-profile-settings.png"><br><br>
     We will use this name in the next step.<br><br>
 
-27. For this step, we'll need to **edit some of the information in the commands** by adding our own information.<br><br>
+26. For this step, we'll need to **edit some of the information in the commands** by adding our own information.<br><br>
```

Comment:
looks like the numbering is off, starting with 28 (outside of the diff, below)

Reply:
fixed numbers [ecb57fe](https://github.com/upleveled/system-setup/pull/91/commits/ecb57fe2e42067fd1de662f9002f6fcaf212d11c)